core/src/main/java/org/jdbi/v3/core/qualifier/QualifiedType.java

```diff
@@ -41,22 +41,16 @@
     private final Set<Annotation> qualifiers;
 
     /**
-     * Creates a QualifiedType for {@code type} with no qualifiers. In practice, the returned object is treated the same
-     * as using {@code type} raw.
-     *
-     * @param type the unqualified type
-     * @return the unqualified QualifiedType
+     * Creates a {@code QualifiedType<T>} for a {@code Class<T>} with no extra qualifiers.
+     * This alone isn't useful; you should call {@link #with} as well.
```

Comment:
This is in fact useful (and in fact necessary), for when you have a `Type` and need an unqualified `QualifiedType`.  For example:

```java
    public Optional<Argument> findFor(Type type, Object value) {
        return findFor(QualifiedType.of(type), value);
    }
```

Reply:
I've made it an `@see` instead